src/client/src/api/fetchWithAuth.js

```diff
@@ -0,0 +1,20 @@
+import store from "../reducers";
+
+export async function fetchWithAuth(url, method, payload = null) {
```

Comment:
REQ: Da diese Methode nur für die neue API v2 verwendet werden kann, sollte man diese entsprechend benennen. Zudem können wir hier eigentlich auch gleich die _Base-Url_ der API v2, d.h. `/api/v2/`, integrieren. So muss diese nicht von jedem Aufrufen wieder mitgegeben werden.

Namensvorschläge: `fetchApiV2`, `fetchApi`..

Die Aufrufe sollten dann in etwa so sein: `await fetchApiV2(`layer?profileId=${profileId}`, "GET");`